src/Request.php

```diff
@@ -66,10 +66,10 @@ public function getRequestTarget(): string
         }
 
         $target = $this->uri->getPath();
-        if ('' == $target) {
+        if ('' === $target) {
             $target = '/';
         }
-        if ('' != $this->uri->getQuery()) {
+        if ('' !== $this->uri->getQuery()) {
```

Comment:
Why Yoda in an expression like this? (Just asking, not looking for a flame war, though IMO this is too much consistency)